### PR TITLE
Enhance bindChart parameters description

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 1]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
 - *"series"* - data fields go to the chart series.
 
 ---
+The **customizeChart**, **customizeSeries** and **processCell** fields of the **integrationOptions** objects are functions. They accept arguments whose structure is described below.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
The **customizeChart**, **customizeSeries**, and **processCell** fields of the **integrationOptions** object are functions. They accept arguments whose structure is described below.
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 2]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+
+- **customizeChart** accepts a single argument:
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
- **customizeChart**
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 3]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        </tr>
+        <tr>
+            <td><b>panes</b></td>
+            <td>Array</td>
+            <td>Allows you to customize the automatically-created panes. Each object in this array can contain fields described in the <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/" target="_blank">panes</a> property description. When this array holds several objects, use their <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/#name" target="_blank">name</a> field to identify the pane.</td>
+        </tr>
+    </table>
+
+    You can also specify any properties (except [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource)) listed in the Chart's [Configuration]/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) section in this object.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
    You can also specify any other [Chart properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/) in this object, except [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource).
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 4]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+
+- **customizeSeries** accepts two arguments:
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
- **customizeSeries** 
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 5]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>axis</b></td>
+            <td>String</td>
+            <td>The name of the axis the series occupies. Has a value only when the chart has multiple axes.</td>
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
            <td>The name of the axis to which the series belongs. Has a value only when the chart has multiple axes.</td>
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 6]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        </tr>
+        <tr>
+            <td><b>pane</b></td>
+            <td>String</td>
+            <td>The name of the pane the series occupies. Has a value only when the chart has multiple panes.</td>
+        </tr>
+    </table>
+
+    You can also specify any properties (except [seriesTemplate](Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/)) listed in the [series]/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) section in this object.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
    You can also specify any other [series properties]/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) in this object, except [seriesTemplate](Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/).
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 7]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+
+- **processCell** accepts a single argument:
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
- **processCell**
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 8]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        </tr>
+        <tr>
+            <td><b>rowPath</b></td>
+            <td>Array</td>
+            <td>The path to the data cell by row field values.</td>
+        </tr>
+        <tr>
+            <td><b>rowPathFormatted</b></td>
+            <td>Array</td>
+            <td>Same as **rowPath**, but with <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#format" target="_blank">format</a> and <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#customizeText" target="_blank">customizeText</a> applied.</td>
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
            <td>Same as <b>rowPath</b>, but with <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#format" target="_blank">format</a> and <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#customizeText" target="_blank">customizeText</a> applied.</td>
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 9]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        </tr>
+        <tr>
+            <td><b>columnPath</b></td>
+            <td>Array</td>
+            <td>The path to the data cell by column field values.</td>
+        </tr>
+        <tr>
+            <td><b>columnPathFormatted</b></td>
+            <td>Array</td>
+            <td>Same as **columnPath**, but with <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#format" target="_blank">format</a> and <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#customizeText" target="_blank">customizeText</a> applied.</td>
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
            <td>Same as <b>columnPath</b>, but with <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#format" target="_blank">format</a> and <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#customizeText" target="_blank">customizeText</a> applied.</td>
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 10]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
 - *"series"* - data fields go to the chart series.
 
 ---
+The **customizeChart**, **customizeSeries**, and **processCell** fields of the **integrationOptions** object are functions. They accept arguments whose structure is described below.
+
+- **customizeChart**
+
+        <!--JavaScript-->customizeChart: function (chartOptions) {
+            // Change chart properties here
+            return chartOptions; // This line is optional
```
Comment (IgnatovDan) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:60]: Imho it should be 

----

PivotGrid code expects that this function will return a dxChart options object (/Documentation/ApiReference/UI_Components/dxChart/Configuration/)

Note:
PivotGrid code will assign new values to 'dataSource ' and 'seriesTemplate' members after this function is executed).

----

See:
- https://github.com/DevExpress/DevExtreme/blob/21_2/js/ui/pivot_grid/ui.pivot_grid.chart_integration.js#L243
- https://github.com/DevExpress/DevExtreme/blob/21_2/js/ui/pivot_grid/ui.pivot_grid.chart_integration.js#L296

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 11]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        }
+        
+    The **chartOptions** object has the following structure:
```
Comment (IgnatovDan) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:63]: The passed object already has values in the following members and  allows you to customize the automatically generated Chart settings

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 12]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+
+- **customizeSeries** 
+
+        <!--JavaScript-->customizeSeries: function (seriesName, seriesOptions) {
+            // Change series properties here
+            return seriesOptions; // This line is optional
+        }
+
+    The **seriesName** argument is a string that allows you to identify a series. The **seriesOptions** argument is an object with the following structure:
+
```
Comment (IgnatovDan) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:93]: The passed object already has values in the following members and allows you to customize the automatically generated Chart settings

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 13]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+            <td>The name of the axis to which the series belongs. Has a value only when the chart has multiple axes.</td>
+        </tr>
+        <tr>
+            <td><b>pane</b></td>
+            <td>String</td>
+            <td>The name of the pane the series occupies. Has a value only when the chart has multiple panes.</td>
+        </tr>
+    </table>
+
+    You can also specify any other [series properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) in this object, except [seriesTemplate](Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/).
```
Comment (IgnatovDan) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]: Imho it should be 

----

PivotGrid code expects that this function will return a [series properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) object

Comment (IgnatovDan) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]: except [seriesTemplate] - no, see https://github.com/DevExpress/DevExtreme/blob/21_2/js/ui/pivot_grid/ui.pivot_grid.chart_integration.js#L260

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 14]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+
+- **processCell**
+
+        <!--JavaScript-->processCell: function (cellData) {
+            // Process data here
+            return cellData; // This line is optional
+        }
+        
+    The **cellData** object has the following structure:
```
Comment (IgnatovDan) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:121]: no, see https://github.com/DevExpress/DevExtreme/blob/21_2/js/ui/pivot_grid/ui.pivot_grid.chart_integration.js#L119

        processCellArgs.chartDataItem = {
            val: value === undefined ? null : value,
            series: seriesName,
            arg: argument
        };

        processCellArgs = processDataCell(processCellArgs, mapOptions.processCell);

**Review by IgnatovDan — CHANGES_REQUESTED** [review]
pls check it

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 15]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>rowPath</b></td>
+            <td>Array</td>
+            <td>The path to the data cell by row field values.</td>
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:132]: Do multiple field values correspond to one data cell? Or do you mean:

`The path to the data cell by row field value.`

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 16]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        </tr>
+        <tr>
+            <td><b>rowFields</b></td>
+            <td>Array</td>
+            <td>Contains objects that are row <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/" target="_blank">fields</a>.</td>
+        </tr>
+        <tr>
+            <td><b>maxRowLevel</b></td>
+            <td>Number</td>
+            <td>A zero-based index that indicates the maximum currently expanded row level. Equals to 0 when the pivot grid has no row fields, 1 when all rows are collapsed, 2 when at least one first-level row is expanded, etc.</td>
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]: 1) `A zero-based index that indicates the maximum currently expanded row level.`
What do you mean by "maximum currently expanded"? For example, do you mean:

`A zero-based index that indicates the maximum number of expanded row levels.`

2) `Equals to 0 when the pivot grid has no row fields, 1 when all rows are collapsed, 2 when at least one first-level row is expanded, etc.`

Please change this to:

`Equals 0 when the pivot grid has no row fields, 1 when all rows are collapsed, 2 when at least one first-level row is expanded, etc.`

or

`Is equal to 0 when the pivot grid has no row fields, 1 when all rows are collapsed, 2 when at least one first-level row is expanded, etc.`

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 17]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        </tr>
+        <tr>
+            <td><b>columnPath</b></td>
+            <td>Array</td>
+            <td>The path to the data cell by column field values.</td>
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:152]: See my similar comment above. Do you mean the following?

`The path to the data cell by column field value.`

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 18]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        </tr>
+        <tr>
+            <td><b>columnFields</b></td>
+            <td>Array</td>
+            <td>Contains objects that are column <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/" target="_blank">fields</a>.</td>
+        </tr>
+        <tr>
+            <td><b>maxColumnLevel</b></td>
+            <td>Number</td>
+            <td>A zero-based index that indicates the maximum currently expanded column level. Equals to 0 when the pivot grid has no column fields, 1 when all columns are collapsed, 2 when at least one first-level column is expanded, etc.</td>
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]: See my similar comments above.

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 19]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
...
+        </tr>
+        <tr>
+            <td><b>dataFields</b></td>
+            <td>Array</td>
+            <td>Contains objects that are data <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/" target="_blank">fields</a>.</td>
+        </tr>
+        <tr>
+            <td><b>dataValues</b></td>
+            <td>Array</td>
+            <td>Data of the current cell. Contains several values if the pivot grid has several data fields.</td>
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]:
```suggestion
            <td>Data of the current cell. Contains several values if the pivot grid has multiple data fields.</td>
```

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md [hunk 20]
```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
 - *"series"* - data fields go to the chart series.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:52]: Can you please make the following changes to the lines above?

Line 20:<br>
Specifies whether to alternate data fields in the resulting chart or not.<br>
`Specifies whether to alternate data fields in the resulting chart.`

Line 31:<br>
Accepts one of the following values.<br>
`Accepts one of the following values:`

Line 48:<br>
Accepts one of the following values.
`Accepts one of the following values:`

Line 50:<br>
`- *"args"* - data fields go to the chart arguments.`
What do you mean by "go to"? For example, do you mean "data fields are the chart arguments"?

Line 51:
`- *"series"* - data fields go to the chart series.`
See my comment above.

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 10 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:60]: > PivotGrid code expects that this function will return a dxChart options object (/Documentation/ApiReference/UI_Components/dxChart/Configuration/)

Here's a pen in which I added an onPointClick event handler to chartOptions _without_ returning chartOptions from customizeChart. As you can see, it works:

https://codepen.io/romantsukanov/pen/RwgqJZd?editors=0010

> PivotGrid code will assign new values to 'dataSource ' and 'seriesTemplate' members after this function is executed).

Which ultimately means that you can't change dataSource and seriesTemplate in these customization functions, and this is exactly what we say.

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 11 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:63]: And?

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 14 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:121]: I'm sorry but I don't understand what you're trying to say. I checked the structure, and it corresponds to the description. You can check it yourself: https://codepen.io/romantsukanov/pen/yLXQEjb?editors=0010

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 13 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]: See my reply to [this comment](https://github.com/DevExpress/devextreme-documentation/pull/2884/files#r713782923).

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 10 above)
Comment (IgnatovDan) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:60]: Return or not return....
Our internal code allows both: chartOptions = extend(true, {}, chartOptions, customizeChart(chartOptions));
It's up to you how to describe it better.

'You can also specify any other Chart properties<link to chart properties> in this object' - this is what our code allows, fine text.

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 13 above)
Comment (IgnatovDan) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]: Again, our code allows both 'return' and 'modify': seriesOptions = extend(seriesOptions, customizeSeries(seriesName, seriesOptions));
The text is up to you.

>The seriesName argument is a string that allows you to identify a series. The seriesOptions argument is an object with the following structure:

fine.

>You can also specify any other series properties in this object, except seriesTemplate.

https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/ - the passed object doesn't contain the 'seriesTemplate' member.
This note of the 'customizeSeries' callback is not correct for the 'seriesOptions' argument.
This note should be in the 'customizeChart' section:

    if(customizeChart) {
        chartOptions = extend(true, {}, chartOptions, customizeChart(chartOptions));
    }

    chartOptions.dataSource = createChartDataSource(dataSource, options, axisDictionary);

    chartOptions.seriesTemplate = {

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 14 above)
Comment (IgnatovDan) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:121]: Sorry, I checked wrong code and my comment is not correct.

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 11 above)
Comment (IgnatovDan) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:63]: These members already have values when customizeChart is called while other members of https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxChart/Configuration/ are 'undefined'.
It's up to you how to describe it better.
Currently it is 'The chartOptions object has the following structure ... ' + 'You can also specify any other Chart properties in this object'

**Review by IgnatovDan — APPROVED** [review]

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 15 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:132]: No, there can be several field values that form the path.

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 20 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:52]: > What do you mean by "go to"?

These putDataFieldsInto property, which can have the values "args" and "series", has to do with conversion of PivotGrid cells and layout into chart series points. By "data fields go to chart arguments", we mean that PivotGrid data fields will be used to create chart arguments. How about

Data fields constitute chart series?

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 16 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]: > What do you mean by "maximum currently expanded"? 

The lowest expanded row level.

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 15 above)
Comment (albertov05) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:132]: Okay, thank you.

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 16 above)
Comment (albertov05) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md]: > The lowest expanded row level.

Thanks for the clarification. Can we say it this way instead?

`A zero-based index that indicates the lowest expanded row level.`

File: api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md (hunk 20 above)
Comment (albertov05) [reply api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md:52]: Sounds good. Please apply your suggestion to lines 50 and 51.

**Pull request merged**